vembrane/modules/filter.py

```diff
@@ -62,46 +62,89 @@ def add_subcommmand(subparsers):
         help="Keep all annotations of a variant if at least one of them passes "
         "the expression.",
     )
+    parser.add_argument(
+        "--preserve-order",
+        default=False,
+        action="store_true",
+        help="Make sure that the variant order of the output matches that of the \
+              input. For performance reasons only make use of this option, if the \
+              input might contain breakends (BNDs) as the order of all other variants \
+              are preserved anyway.",
+    )
+
+
+def test_and_update_record(env, idx, record, ann_key, keep_unmatched):
```

Comment:
I think we should have type info here as well, especially since the return type is now a tuple of things?

Reply:
added some type infos